docs/shared/using-nx/affected.md

```diff
@@ -6,7 +6,7 @@ Before reading this guide, [check out the mental model guide](/concepts/mental-m
 
 ## Overview
 
-When you run `nx test app1`, you are telling Nx to run the app1:test task plus all the tasks it depends on.
+When you run `nx test app1`, you are telling Nx to run the app1:test task plus all the tasks they depend on.
```

Comment:
This should stay `it depends`.

There is a single project depending on the libraries.